packages/react-components/src/components/chart/utils/getXAxis.ts

```diff
@@ -19,6 +19,6 @@ export const getXAxis = (viewportInMs: ViewportInMs, axis?: ChartAxisOptions): X
     },
     splitNumber: 6,
     min: 0,
-    max: viewportInMs.isDurationViewport ? viewportInMs.end : undefined,
```

Comment:
undefined was leaving the x axis max to be auto calculated
we were running into the situation where using the time machine to select a new range made it impossible to scroll to the right :( 

setting the max to be date.now allows a user to select a date in the past and still scroll left/right as expected